print_env.sh

```diff
@@ -0,0 +1,49 @@
+# Reports relevant environment information useful for diagnosing and
+# debugging cuDF issues.
+# Usage: 
+# "./print_env.sh" - prints to stdout
+# "./print_env.sh > env.txt" - prints to file "env.txt"
+
+echo "***git log --decorate -n 1***"
```

Comment:
Don't echo what you're about to run. Put `set -x` at the top and it will do it for you.